plugins.json

```diff
@@ -524,5 +524,14 @@
 		"version": "0.0.1",
 		"variant": "both",
 		"about": "This plugin simply adds a button that rearranges your individual Per-face UV rectangles in the same way Box-UV mode would. This is handy if you want to use the standard Box-UV format, but also have the flexibility of Per-face UV mode."
+	},
+	"animation_to_java": {
+		"title": "Animation to Java Converter",
+		"author": "MG",
+		"description": "Converts Blockbench animations to Java code for the new 1.19 keyframe system",
+		"icon": "fa-cube",
+		"version": "1.0.0",
+		"variant": "both",
```

Comment:
Meta data must be the same in plugins.json and in the plugin itself. Here the tags are missing.